src/lib.rs

```diff
@@ -79,6 +79,14 @@ pub struct Command {
     id_map_commands: Option<(PathBuf, PathBuf)>,
 }
 
+impl Drop for Command {
+    fn drop(&mut self) {
+        for (_, fd) in &self.config.setns_namespaces {
+            unsafe { libc::close(*fd) };
```

Comment:
To clarify: I mean a wrapper around file descriptor. Similarly to how it's done for `unshare::stdio::Fd`